Controller/ChangePasswordController.php

```diff
@@ -31,7 +31,7 @@
  * @author Thibault Duplessis <thibault.duplessis@gmail.com>
  * @author Christophe Coevoet <stof@notk.org>
  */
-class ChangePasswordController extends Controller
+class ChangePasswordController extends AbstractController
```

Comment:
I think this is a BC break for projects extending the class. And we have not marked it as `@final` for now (which we should, as we did such BC break in the past).
So this should at least be documented in the upgrade file.

Reply:
@stof  So this is what we could do: 

- we don't do the update, but we release a minor version, including the @final tag
- I don't know it there is a major version currently in the pipe. But it could be the occasion. For instance, it could also include some code cleanup to take advantage of the autowiring in S4? What do you think?

Reply:
Hello @stof Any news on this matter?

Reply:
There is no new major version in the pipeline.

As @stof mentioned, we can mark the controllers @final as we did in the past.

Reply:
@enrico69 Sure, we could do that.